pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+    </ol>
+  </div>
+  
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a5"
+    >
+      If you have code older than USWDS 2.10.1
+    </button>
+  </h4>
+  <div id="m-a5" class="usa-accordion__content usa-prose">
+    <p><strong><span class="usa-tag bg-accent-warm-darker">Settings</span> We updated some settings defaults:</strong></p>
+    <ul>
+      <li>
+        <strong>Updated: </strong>$theme-breadcrumb-background-color: default<br>
+        <strong>Was: </strong>white 
+      </li>
+      <li>
+        <strong>Updated: </strong>$theme-alert-icon-size: 5 <br>
+        <strong>Was: </strong>4
+      </li>
+    </ul>
+    <p><strong>What to do:</strong></p>
+    <ol>
+      <li>Search your codebase for any instances of <strong>the old theme</strong></li>
+      <li>Replace it with <strong>the new setting</strong></li>
+    </ol>
+    <p><strong><span class="usa-tag bg-accent-cool-darker">Markup</span> We updated usa-footer__logo-heading to use a p instead of an h3.</strong> We improved the accessibility of the footer by converting a non-semantic heading into paragraph text.</p>
+    <p><strong>What to do: </strong></p>
+    <ol>
+      <li>TK</li>
+      <li>TK</li>
+    </ol>
+  </div>
+</div>
+
+### 3. Install the USWDS 3.0 package
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+1. In a terminal window, navigate to your project root. 
+2. Install USWDS 3.0 with **npm install @uswds/uswds --save --save-exact**
+3. Uninstall USWDS 2 with **npm uninstall uswds**
+
+### 4. Update your Sass compiler settings and recompile CSS
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+The location of the USWDS source files is different in USWDS 3.0. You'll need to update your compiler settings to compile your Sass from USWDS 3.0.
+
+<div class="usa-accordion usa-accordion--bordered">
+  <!-- Use the accurate heading level to maintain the document outline -->
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a6"
+    >
+      If you're using USWDS Gulp
+    </button>
+  </h4>
+  <div id="m-a6" class="usa-accordion__content usa-prose">
+    <ol>
+      <li>Search for a line like <strong>const USWDS = "node_modules/uswds/dist"</strong> in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.</li>
+      <li>
+        Update the USWDS <strong>const</strong> to the updated uswds package location
+        <p><strong>Old</strong></p>
+          <code>const USWDS = "node_modules/uswds/dist";</code>
+        <p><strong>New</strong></p>
+          <code>const USWDS = "node_modules/@uswds/uswds";</code>
+          <br>
+      </li>
+      <li> 
+        Search for <strong>includePaths</strong> in your project's Gulp files. The paths in this list are where the Sass compiler looks for your source files. In USWDS 3.0
```

Comment:
@mahoneycm Missing the rest of this sentence?`In USWDS 3.0`

Reply:
Hmm that's a @bonnieAcameron question. That's how it is in the mockup but I assume maybe we can just delete the `In USWDS 3.0`

![image](https://user-images.githubusercontent.com/25211650/164545243-65170d80-7e99-4a0b-bd35-82d26807306b.png)

Reply:
Delete it!

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/1529/commits/ef0d34faf7748099236067b3f2e1b51f60ac0493